seed/lib/uniformat/uniformat.py

```diff
@@ -0,0 +1,4418 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+SEED Platform (TM), Copyright (c) Alliance for Sustainable Energy, LLC, and other contributors.
+See also https://github.com/seed-platform/seed/main/LICENSE.md
+"""
+from django.db import transaction
+from typing_extensions import NotRequired, TypedDict
+
+from seed.models import Uniformat
+
+
+class UniformatEntry(TypedDict):
+    code: str
+    category: str
+    definition: NotRequired[str]
+    imperial_units: NotRequired[str]
+    metric_units: NotRequired[str]
+    quantity_definition: NotRequired[str]
+
+
+uniformat_data: list[UniformatEntry] = [
+    {
+        'code': 'A',
+        'category': 'SUBSTRUCTURE',
+        'definition': 'This system includes all work below the lowest floor construction (usually slab-on-grade) and the enclosing horizontal and vertical elements required to form a basement, together with the necessary mass excavation and backfill.',
+        'imperial_units': 'SF',
```

Comment:
this is fine for now, but we should use the format that is defined in pint (e.g., ft**2)

Reply:
I'm happy to change it, everything is verbatim from https://www.wbdg.org/ffc/navy-navfac/design-build-request-proposal/uniformat-structure